en/SUMMARY.md

```diff
@@ -611,7 +609,7 @@
   * [ROS](ros/README.md)
     * [ROS 2](ros/ros2.md)
       * [ROS 2 User Guide](ros/ros2_comm.md)
-      * [ROS 2 microRTPS Offboard Control Example](ros/ros2_offboard_control.md)
+      * [ROS 2 microRTPS Offboard Control Example](ros/ros2_offboard_control.md) 
```

Comment:
Note the title. Touched this because we will need to rewrite this example for the new architecture.